main/tests/server/src/com/google/refine/io/ProjectMetadataUtilitiesTest.java

```diff
@@ -0,0 +1,56 @@
+
+package com.google.refine.io;
+
+import com.google.refine.ProjectMetadata;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class ProjectMetadataUtilitiesTest {
+
+    ProjectMetadata actualMetadata = new ProjectMetadata();
+    ProjectMetadata expectedMetadata = new ProjectMetadata();
+    File file = null;
+
+    @Before
+    public void setUp() {
+        actualMetadata.setName("Velmi dobře, použijeme vícejazyčný neurální stroj, abychom to vyřešili. ÜÜÖÖÄÄ");
+
+        file = new File("tests/server/src/com/google/refine/io/example_project_metadata_charset_test.json");
```

Comment:
It would be better to use a temporary file instead (files created in this location will not be cleaned by `mvn clean`).
You should be able to use `File.createTempFile` for that.

Reply:
🤔 true! Alright, thanks.